packages/dev/loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -1509,9 +1509,8 @@ export class GLTFLoader implements IGLTFLoader {
             channel.target.extensions = channel.target.extensions || {};
             channel.target.extensions.KHR_animation_pointer = {
                 pointer: `/nodes/${channel.target.node}/${channel.target.path}`,
+                lazzy: true,

```

Comment:
This isn't much better than the original way. Let's just go with the original way for now. We can deal with this later if there is an issue.